Server/mods/deathmatch/logic/CMainConfig.cpp

```diff
@@ -232,43 +247,37 @@ bool CMainConfig::Load()
     GetInteger(m_pRootNode, "verifyclientsettings", m_iEnableClientChecks);
 
     // Handle the <client_file> nodes
-    CXMLNode*    pNode = NULL;
-    unsigned int uiCurrentIndex = 0;
-    do
-    {
-        // Grab the current script node
-        pNode = m_pRootNode->FindSubNode("client_file", uiCurrentIndex++);
-        if (pNode)
+    currentNode = nullptr;
+    currentIndex = 0;
+    while (currentNode = m_pRootNode->FindSubNode("client_file", currentIndex++))
+    {
+        // Grab its "name" attribute
+        CXMLAttribute* attribute = currentNode->GetAttributes().Find("name");
+        SString name = attribute ? attribute->GetValue() : SString{};
+        name = name.Replace("\\", "/").ToLower();
+
+        // Grab its "verify" attribute
+        attribute = currentNode->GetAttributes().Find("verify");
+        SString verify = attribute ? attribute->GetValue() : SString{};
+        bool shouldVerify = verify == "true" || verify == "yes" || verify == "1";
+
+        // Find bitnumber
+        bool found = false;
+        for (std::uint8_t i = 0; i < NUMELMS(gtaDataFiles); i++)
```

Comment:
std::size_t. And since you deciced to refactor the code and Deathmatch project is C++17, you can use std::size instead of NUMELMS.